bin/jsii-release-npm

```diff
@@ -38,7 +38,7 @@ fi
 log=$(mktemp -d)/npmlog.txt
 
 for file in ${dir}/**.tgz; do
-  npm publish ${tag} ${file} 2>&1 | tee ${log}
+  npm publish ${tag} "${NPM_PUBLISH_ARGS}" ${file} 2>&1 | tee ${log}
```

Comment:
What happens if this env variable is not defined?

Reply:
It should fall back to an empty string without impact. But I might be wrong about this.

Maybe the quotes should not be there

Reply:
Personally I am fan of non-leaky abstractions so I'd go with explicit support for this feature.

Please make sure you manually test this with and without this option since there are no tests in this module at the moment.

Reply:
Ok, I will change it.